scripts/spellcheck.py

```diff
@@ -1,11 +1,12 @@
+import csv
 import glob
 import os
-import csv
 import re
 import sys
+from dataclasses import dataclass
 
+from pygit2 import Repository
```

Comment:
Better to just have this be a parameter handled via the Makefile that's changed between branches.